src/callbacks_step/save_solution_dg.jl

```diff
@@ -122,7 +122,7 @@ function save_solution_file(u, time, dt, timestep,
     attributes(file)["equations"] = get_name(equations)
     attributes(file)["polydeg"] = polydeg(dg)
     attributes(file)["n_vars"] = n_vars
-    attributes(file)["n_elements"] = nelements(dg, cache)
+    attributes(file)["n_elements"] = cache.mpi_cache.n_elements_global
```

Comment:
Would it make sense to introduce another function returning the global number of elements?

Reply:
@sloede ?

Reply:
It depends on how often this is used. If it is just read once or twice, I don't think it is required right now. OTOH... having `nelementsglobal(dg cache)` would give me a nice and warm feeling about consistency \o/, so why not?

Reply:
What would be the best way to do this? This function only makes sense for parallel execution and `mpi_cache` is also only defined in parallel execution.

Reply:
Maybe something like
```julia
nelementsglobal(dg, cache) = mpi_isparallel() ? cache.mpi_cache.n_elements_global : nelements(dg, cache)
```
Havent tried it though, so maybe there's still an error in the code above.

Reply:
Looks good, thank you!